docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+services:
```

Comment:
maybe add a open-ldap container for testing with

Reply:
Does that need to be added for testing?

Reply:
if you want to test all the write and update functions not on the main ldap yes, if you have a testing ldap then no